components/Timeline.tsx

```diff
@@ -0,0 +1,43 @@
+// *Imports
+import React from "react";
+import { Text, View } from "react-native";
+
+// *Constants
+const days = ["Sun", "Mon", "Tue", "Wed", "Thu", "Fri", "Sat"];
+const today = new Date();
+const todayIndex = today.getDay();
+
+// *Start of the Timeline Function
+export default function Timeline() {
```

Comment:
Not sure if this is included in the component feature, but thinking ahead, we could potentially have a selected state and onSelectedStateChange being passed with the default state being that which corresponds to the current date, but when the user clicks into a previous bubble, they would select a new day which would allow them to view the previous feeds.